### PR TITLE
Added Certified Moderator badge flag.

### PR DESCRIPTION
**Pull request by vanishedvan**
**Please describe the changes this PR makes and why it should be merged:**
This pull request adds the CERTIFIED_MODERATOR badge flag to the UserFlags class.

PR in discord-api-docs: https://github.com/discord/discord-api-docs/pull/2946

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by iCrawl** [comment]
We might as well wait until they rename it `EARLY_CERTIFIED_MODERATOR` because people try to abuse it again 😄

**Comment by advaith1** [comment]
add to typings

https://github.com/discordjs/discord.js/blob/master/typings/index.d.ts#L3609-L3621

**Comment by vanishedvan** [comment]
> add to typings
> 
> https://github.com/discordjs/discord.js/blob/master/typings/index.d.ts#L3609-L3621

Done.

File: src/util/UserFlags.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ class UserFlags extends BitField {}
  * * `BUGHUNTER_LEVEL_2`
  * * `VERIFIED_BOT`
  * * `EARLY_VERIFIED_BOT_DEVELOPER`
+ * * `CERTIFIED_MODERATOR`
```
Comment (iShibi) [inline src/util/UserFlags.js:37]:
```suggestion
 * * `DISCORD_CERTIFIED_MODERATOR`
```

File: src/util/UserFlags.js [hunk 2]
```diff
@@ -50,6 +51,7 @@ UserFlags.FLAGS = {
   BUGHUNTER_LEVEL_2: 1 << 14,
   VERIFIED_BOT: 1 << 16,
   EARLY_VERIFIED_BOT_DEVELOPER: 1 << 17,
+  CERTIFIED_MODERATOR: 1 << 18,
```
Comment (iShibi) [inline src/util/UserFlags.js:54]:
```suggestion
  DISCORD_CERTIFIED_MODERATOR: 1 << 18,
```

File: typings/index.d.ts [hunk 3]
```diff
@@ -3618,7 +3618,8 @@ declare module 'discord.js' {
     | 'TEAM_USER'
     | 'BUGHUNTER_LEVEL_2'
     | 'VERIFIED_BOT'
-    | 'EARLY_VERIFIED_BOT_DEVELOPER';
+    | 'EARLY_VERIFIED_BOT_DEVELOPER'
+    | 'CERTIFIED_MODERATOR';
```
Comment (iShibi) [inline typings/index.d.ts:3622]:
```suggestion
    | 'DISCORD_CERTIFIED_MODERATOR';
```

**Review by iShibi — CHANGES_REQUESTED** [review]
This time they [changed](https://github.com/discord/discord-api-docs/pull/2946#discussion_r634771408) the name too quickly 🏃‍♀️ 💨

**Comment by Lulalaby** [comment]
> 
> 
> We might as well wait until they rename it `EARLY_CERTIFIED_MODERATOR` because people try to abuse it again 😄

As I know, not every DMD member gets it, you have to earn it and it'll be manually assigned by discord staff.

**Comment by iCrawl** [comment]
image

**Review by iCrawl — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**